src/Set/PHPUnitSetList.php

```diff
@@ -63,6 +63,11 @@ final class PHPUnitSetList implements SetListInterface
      */
     public const PHPUNIT_CODE_QUALITY = __DIR__ . '/../../config/sets/phpunit-code-quality.php';
 
+    /**
+     * @var string
+     */
+    public const PHPUNIT_CODING_STYLE = __DIR__ . '/../../config/sets/phpunit-coding-style.php';
```

Comment:
Let's keep this simple and have only the code quality set. There is already enough tiny sets and I want to avoid in choice paralaysis.